doc/rust.txt

```diff
@@ -157,6 +157,14 @@ COMMANDS                                                       *rust-commands*
 		If |g:rustc_path| is defined, it is used as the path to rustc.
 		Otherwise it is assumed rustc can be found in $PATH.
 
+:Play                                                                   *Play*
```

Comment:
Oops! Fixed.

On Thu, Mar 26, 2015 at 5:40 PM, Chris Morgan <notifications@github.com>
wrote:

> In doc/rust.txt
> <https://github.com/rust-lang/rust.vim/pull/16#discussion_r27269424>:
>
> > @@ -157,6 +157,14 @@ COMMANDS                                                       *rust-commands*
> >  		If |g:rustc_path| is defined, it is used as the path to rustc.
> >  		Otherwise it is assumed rustc can be found in $PATH.
> >
> > +:Play                                                                   *Play*
>
> *Play* should be *:Play*.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust.vim/pull/16/files#r27269424>.
>